backend/api/v2beta1/pipeline.proto

```diff
@@ -95,6 +95,14 @@ service PipelineService {
     };
   }
 
+  // Creates a new pipeline and a new pipeline version in a single transaction.
+  rpc CreatePipelineAndVersion(CreatePipelineAndVersionRequest) returns (Pipeline) {
```

Comment:
Given that we have the test infra now, when adding new API endpoint, we should also add the integration test, either in 
https://github.com/kubeflow/pipelines/blob/master/backend/test/integration/pipeline_api_test.go
or https://github.com/kubeflow/pipelines/blob/master/backend/test/integration/pipeline_version_api_test.go

Reply:
Sounds good. Let me fix it.